c2corg_ui/static/js/documentdetailsview.js

```diff
@@ -68,6 +89,47 @@ app.ViewDetailsController.prototype.openModal = function(selector) {
   this.modal_.open({animation: true, size: 'lg', template: template});
 };
 
+
+/**
+ * @param {Date} dateStart
+ * @param {Date} dateEnd
+ * @suppress {checkTypes} it told (found scope | null, required -> scope) at compile()(this.scope)...
+ * @export
+ */
+app.ViewDetailsController.prototype.displayDate = function(dateStart, dateEnd) {
+
+  //angular executes this function so many times it tells 'Error: 10 $digest() iterations reached. Aborting' so
+  // with this quick n dirty method there is no error. Any idea how to do it in the pro way ?
```

Comment:
We have to find why this function is called many times!
It seems to be called only once in this expression: 
https://github.com/c2corg/v6_ui/blob/all-view-fixes-2/c2corg_ui/templates/outing/view.html#L31
```
<span class="outing-date" class="ng-cloak">{{detailsCtrl.displayDate('${outing['date_start']}', '${outing['date_end']}') }}</span>
```
Can't we have a one-time binding?
Anyway do we really need angular here? Why not using a Mako def to do this in python?